test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -392,15 +387,12 @@ func benchmarkScheduling(numNodes, numExistingPods, minPods int,
 	b.ResetTimer()
 	config = testutils.NewTestPodCreatorConfig()
 	config.AddStrategy("sched-test", b.N, testPodStrategy)
-	podCreator = testutils.NewTestPodCreator(c, config)
+	podCreator = testutils.NewTestPodCreator(clientset, config)
 	podCreator.CreatePods()
 	for {
 		// This can potentially affect performance of scheduler, since List() is done under mutex.
 		// TODO: Setup watch on apiserver and wait until all pods scheduled.
-		scheduled, err := schedulerConfigFactory.GetScheduledPodLister().List(labels.Everything())
-		if err != nil {
-			klog.Fatalf("%v", err)
-		}
+		scheduled := schedulerConfig.SchedulingQueue.PendingPods()
```

Comment:
Are pending pods equivalent to scheduled pods?

Reply:
+1. 
The [PendingPods](https://github.com/kubernetes/kubernetes/blob/315f34bb5146a4679ee800e7948da0f196a4b460/pkg/scheduler/internal/queue/scheduling_queue.go#L608-L610) comment says it's "used for debugging purposes in the scheduler cache dumper and comparer". I wonder whether the comment is inaccurate or it's not supposed to be used here.

Reply:
Sorry for accidentally resolve this conversation, I'll dig into this and find the proper way for this.

Reply:
This problem has been solved

/hold cancel